gazelle/README.md

```diff
@@ -61,7 +57,7 @@ gazelle_python_manifest(
     name = "gazelle_python_manifest",
     # The @modules_map refers to the name we gave in the modules_mapping
     # rule in the WORKSPACE
```

Comment:
Reference to the old repository name.

Reply:
Fixed in https://github.com/bazelbuild/rules_python/pull/578/commits/fa6a7cf2f3b11eed1a0e382a0c1823f2d7637cfe.